### PR TITLE
Make multi-tenant support and release namespace optional

### PR DESCRIPTION
**Pull request by deejgregor**
- Add releaseNamespace option that controls which namespace to use
- Make multi-tenant support optional and disabled by default.

### CODE REVIEW

File: horizon/values.schema.json [hunk 1]
```diff
@@ -279,6 +279,16 @@
             "title": "Create Namespace",
             "default": false
         },
+        "releaseNamespace": {
+            "type": "boolean",
+            "title": "Put the objects in a namespace named for the release"
```
Comment (mershad-manesh) [inline horizon/values.schema.json:284]: missing `,` at the end of the line

File: horizon/templates/app-settings.configmap.yaml [hunk 2]
```diff
@@ -56,7 +64,12 @@ data:
   GF_SERVER_ROOT_URL: "/"
   {{- end }}
   GF_DATABASE_TYPE: "postgres"
+  {{- if .Values.multiTenant }}
   GF_DATABASE_NAME: {{ printf "%s_grafana" .Release.Name | quote }}
+  {{- else }}
+  GF_DATABASE_NAME: "grafana"
+  {{- else }}
```
Comment (mershad-manesh) [inline horizon/templates/app-settings.configmap.yaml:71]: We might want to remove the extra `else`

File: horizon/values.schema.json (hunk 1 above)
Comment (deejgregor) [reply horizon/values.schema.json]: Ah, thanks.

File: horizon/templates/app-settings.configmap.yaml (hunk 2 above)
Comment (deejgregor) [reply horizon/templates/app-settings.configmap.yaml]: Yup. :)

File: horizon/templates/opennms-core.clusterrolebinding.yaml [hunk 3]
```diff
@@ -6,7 +6,7 @@ metadata:
 subjects:
   - kind: ServiceAccount
     name: {{ (.Values.dependencies.securitycontext).serviceaccount.name | quote }}
-    namespace: {{ .Release.Name }}
+    {{ include "namespace" . }}
```
Comment (mershad-manesh) [inline horizon/templates/opennms-core.clusterrolebinding.yaml:9]: missing `namespace: `

**Pull request merged**